tests/test_framework/revaultd.py

```diff
@@ -33,18 +33,18 @@ def __init__(
 
         # The data is stored in a per-network directory. We need to create it
         # in order to write the Noise private key
-        datadir_with_network = os.path.join(datadir, "regtest")
-        os.makedirs(datadir_with_network, exist_ok=True)
+        self.datadir_with_network = os.path.join(datadir, "regtest")
```

Comment:
bbf3e14d02d35b8d44558fa120be25bf5aba3ef5 title: it's an object attribute, not a class one :)